test/functional/pdo_sqlsrv/pdo_transaction.phpt

```diff
@@ -3,51 +3,69 @@ test transaction rollback and commit
 --DESCRIPTION--
 starts a transaction, delete rows and rollback the transaction; starts a transaction, delete rows and commit
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-    require_once("MsSetup.inc");
-
-    $conn = new PDO( "sqlsrv:Server=$server; database = $databaseName", $uid, $pwd);
-    
-    $conn->exec("IF OBJECT_ID('Table1', 'U') IS NOT NULL DROP TABLE Table1");
-    $conn->exec("CREATE TABLE Table1(col1 CHARACTER(1), col2 CHARACTER(1))");
-   
-    $ret = $conn->exec("INSERT INTO Table1(col1, col2) VALUES('a', 'b')");
-    $ret = $conn->exec("INSERT INTO Table1(col1, col2) VALUES('a', 'c')");
-   
+require_once("MsCommon_mid-refactor.inc");
+
+function deleteRows($conn, $tbname)
+{
+    if (!isColEncrypted()) {
+        $rows = $conn->exec("DELETE FROM $tbname WHERE col1 = 'a'");
+    } else {
+        // needs to find parameter for encrypted columns
+        $sql = "DELETE FROM $tbname WHERE col1 = ?";
+        $stmt = $conn->prepare($sql);
+        $col1 = "a";
+        $stmt->execute(array($col1));
+        $rows = $stmt->rowCount();
+    }
```

Comment:
This assumes the delete will always succeed? No exception handling?

Reply:
there is exception handling in main